src/fiat-connect-client.ts

```diff
@@ -82,9 +83,18 @@ export class FiatConnectClient implements FiatConnectApiClient {
    * @returns a Promise resolving to the literal string 'success' on a
    * successful login or an Error response.
    */
-  async login(): Promise<Result<'success', ResponseError>> {
+  async login(params: LoginParams): Promise<Result<'success', ResponseError>> {
     try {
-      const expirationDate = new Date(Date.now() + SESSION_DURATION_MS)
+      // Prefer param expiration time > diff-based exp. time > client-based exp. time
+      let expirationDate = params?.expirationDate
```

Comment:
what about the issuedAt time? Doesn't that need to use the server's time also?

Reply:
+1, if we just adjust the issuedAt time based on clock diff, we could just use issuedAt + 4h for expiration time instead of adjusting it separately?

Reply:
Yeah, this is a lot cleaner IMO. I've updated the implementation to accept an `issuedAt` time and simplified the helper to just get the earliest approximation of the server's current clock.